internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
```

Comment:
The handlers such as `ListCommitsInPullRequest`, `ListPatchesInPullRequest`, and `GenerateCodeReview` might face issues if they do not validate the path parameters. Ensure that your functions validate the parameters (`repo`, `pull_number`, `owner`) before attempting to use them, to avoid potential panics or unexpected behavior. Implement validation checks and return appropriate HTTP responses for invalid inputs.